ProjetSupaero2018/catkin_ws/src/display/scripts/G_Display.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+import rospy
+import tf
+from rospy_tutorials.msg import Floats
+from display.msg import Control
+from gazebo_msgs.srv import SpawnModel
+from geometry_msgs.msg import Point, Quaternion, Pose
+
+_car_name = 'my_car'
+
+
+def callbackJalon2(data):
```

Comment:
Is it really a function name? (and is it PEP8 compliant, btw? I thought syntax would be callback_jalon_2)